### DISCUSSION TITLE
Why Pocketbase exist? What problems it want to solve?

### DISCUSSION THREAD

**Discussion by rubiagatra** [pocketbase/pocketbase#3760] [Q&A]
Just wondering what problems Pocketbase wants to solve. And maybe we can put It on documentation/readme @ganigeorgiev?

**Comment by ganigeorgiev** [comment]
There is already note in the README footer. In short it was created to assist with my other open source project Presentator. For more details why you can read in https://github.com/presentator/presentator/issues/183.
